invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/landing_page/ShareOptions/AccessUsers/AccessUsers.js

```diff
@@ -61,13 +61,12 @@ export class AccessUsers extends Component {
   render() {
     const { record, handleClose, permissions } = this.props;
     const { results, loading, error } = this.state;
-
     return (
       <>
         {error && error}
         <Modal.Content>
           {loading && <Loader isLoading active />}
-          {!loading && !isEmpty(results) && (
+          {!loading && results != undefined && (
```

Comment:
just for me to understand what happened: why not to use `isEmpty`?

Reply:
Because when the grants list is empty we still should see the owner and the `Add people` button  <img width="1229" alt="Screenshot 2024-04-09 at 10 16 33" src="https://github.com/inveniosoftware/invenio-app-rdm/assets/61321254/61c68b02-4c1e-483d-a9f7-fe297ad951ee">